src/main/java/pl/project13/maven/git/CommitIdGenerationMode.java

```diff
@@ -14,13 +14,9 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with git-commit-id-plugin.  If not, see <http://www.gnu.org/licenses/>.
  */
+package pl.project13.maven.git;
 
-package pl.project13.maven.git.log;
-
-public interface LoggerBridge {
```

Comment:
Let's keep the bridge for now, and back it with the Mojo; such that we make it easier to rip out the core without depending on maven (the Mojo). Thanks!